features/storage/bd/SDBlockDevice.cpp

```diff
@@ -112,22 +112,30 @@
  * | 0xFE | data[0] | data[1] |        | data[n] | crc[15:8] | crc[7:0] |
  * +------+---------+---------+- -  - -+---------+-----------+----------+
  */
-#include "SDFileSystem.h"
+#include "SDBlockDevice.h"
 #include "mbed_debug.h"
 
 #define SD_COMMAND_TIMEOUT 5000
 
-#define SD_DBG             0
+#define SD_DBG             1
```

Comment:
is this better enabled or disabled by default? I'd hate people not to see the debug member and change this to turn it off and fork the library